<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="Main_widget">
   <widget class="QGroupBox" name="DOF">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>601</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Degrees of Freedom</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>481</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,3,0,3,0,3">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>25</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="x_label">
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="x_dsb"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="y_label">
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="y_dsb"/>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="z_label">
        <property name="text">
         <string>Z:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QDoubleSpinBox" name="z_dsb"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="roll_label">
        <property name="text">
         <string>Roll:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="roll_dsb"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="pitch_label">
        <property name="text">
         <string>Pitch:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="pitch_dsb"/>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="yaw_label">
        <property name="text">
         <string>Yaw:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QDoubleSpinBox" name="yaw_dsb"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Buttons">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>591</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Buttons</string>
    </property>
    <widget class="QCheckBox" name="left_cb">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Left</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="right_cb">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Right</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="device_groupbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>591</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Connexion Device:</string>
    </property>
    <widget class="QComboBox" name="device_cb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>571</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>647</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
